source/includes/v2/_dataset.md

```diff
@@ -70,177 +78,198 @@ curl 'https://documentation-resources.opendatasoft.com/api/v2/catalog/datasets/d
 ```shell
 curl 'https://documentation-resources.opendatasoft.com/api/v2/catalog/datasets/doc-geonames-cities-5000/records?where="Noa"'
 ```
-
-This endpoint provides a search facility in the dataset catalog.
-
-<aside>
-It is not possible to retrieve all records from a dataset with this API. To do so, export endpoints must be used.
-</aside>
-
-##### HTTP Request
-`GET /api/v2/catalog/datasets/<dataset_id>/records`
-
-##### URL Parameters
-
-Parameter | Default | Description
---------- | ------- | -----------
-`select` | * | Select expression used to retrieve specific fields (see [ODSQL documentation](#select-clause))
-`where` | None | Filter expression used to restrict returned datasets (see [ODSQL documentation](#where-clause))
-`refine` | None | Refine the result set on a given facet value (see [refine in Facet documentation](#refine))
-`exclude` | None | Exclude a given facet value from the result set (see [exclude in Facet documentation](#exclude))
-`offset` | 0 | Index of the first item to return
-`limit` | 10 | Number of items to return
-`include_app_metas` | false | Explicitly request application metadata for each dataset
-`timezone` | UTC | Timezone applied on datetime fields in query and response
-
-<aside>
-The sum of `offset` and `limit` parameters must not exceed 10000. Use the export API to download all records.
-</aside>
-
-
-## Aggregating records
-
-> Aggregation query without group_by
+> Grouping all records of a dataset in order to perform a count
 
 ```shell
-curl 'https://documentation-resources.opendatasoft.com/api/v2/catalog/datasets/doc-geonames-cities-5000/aggregates?select=count(*) as count'
+curl 'https://documentation-resources.opendatasoft.com/api/v2/catalog/datasets/doc-geonames-cities-5000/records?select=count(*) as count&group_by'
```

Comment:
This isolated `group_by` parameter looks a bit weird without more explanation.

Should we illustrate the same query with and without `group_by` ? Or maybe just explain that `group_by` without value means something like "return only one group and compute the aggregation function over all the records" ?